packages/g/glycin/package.yml

```diff
@@ -0,0 +1,30 @@
+name       : glycin
+version    : 0.1.2
+release    : 1
+source     :
+    - https://download.gnome.org/sources/glycin-loaders/0.1/glycin-loaders-0.1.2.tar.xz : c76c01925abd070174589ccb916a44a573ab65b1e9d493f154e42756431e6dd7
+homepage   : https://gitlab.gnome.org/sophie-h/glycin
+license    :
+    - LGPL-2.1-or-later
+    - MPL-2.0
+component  : desktop.gnome
+summary    : Sandboxed and extendable image decoding
+description: |
+    Glycin allows to decode images into gdk::Texture(s) and to extract image metadata. The decoding happens in sandboxed modular image loaders.
+builddeps  :
+    - pkgconfig(cairo)
+    - pkgconfig(gtk4)
+    - pkgconfig(libheif)
+    - pkgconfig(libxml-2.0)
+    - cargo
+    - git
+    - rust
+setup      : |
+    %meson_configure -Dtest_skip_install=true
+build      : |
+    %ninja_build
+install    : |
+    %ninja_install
+check      : |
+    # It always failed
+    # %ninja_check
```

Comment:
Do they succeed if you build them outside of solbuild?

Reply:
Yes, test was ran successfully outside solbuild

Reply:
That is log of test inside  and outside solbuild.
[inside.txt](https://github.com/getsolus/packages/files/13944225/inside.txt)
[outside.txt](https://github.com/getsolus/packages/files/13944223/outside.txt)

Reply:
Maybe you need libpng installed inside solbuild?
`called `Result::unwrap()` on an `Err` value: UnknownImageFormat("image/png")`

Reply:
libpng is deps of libcairo